### ISSUE TITLE
Blank comments in 9.5.1

### ISSUE THREAD

**Issue by mYsn1k** [glpi-project/glpi#7983]
**Describe the bug**

In tickets view there's blank comment page

**Logs**

```
[2020-09-21 11:43:36] glpiphplog.CRITICAL:   *** Uncaught Exception ArgumentCountError: Too few arguments to function _sn(), 2 passed in /var/www/help/inc/commonitilobject.class.php on line 6490 and at least 3 expected in /var/www/help/i
nc/autoload.function.php at line 202
  Backtrace :
  inc/commonitilobject.class.php:6490                _sn()
  inc/commonitilobject.class.php:6521                CommonITILObject->filterTimeline()
  inc/commonitilobject.class.php:6955                CommonITILObject->showTimelineHeader()
  inc/ticket.class.php:801                           CommonITILObject->showTimeline()
  inc/commonglpi.class.php:637                       Ticket::displayTabContentForItem()
  ajax/common.tabs.php:92                            CommonGLPI::displayStandardTab()

```

**Screenshots**
![image](https://user-images.githubusercontent.com/70521919/93749312-61d98180-fc02-11ea-978a-b4600e65fe08.png)

**Your GLPI setup (you can find it in Setup > General menu, System tab)**
```
`
GLPI 9.5.1 ( => /var/www/help) Installation mode: TARBALL


Operating system: Linux glpi 4.15.0-117-generic #118-Ubuntu SMP Fri Sep 4 20:02:41 UTC 2020 x86_64 PHP 7.2.24-0ubuntu0.18.04.6 apache2handler (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, apc, 	apcu, bz2, calendar, ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, imap, intl, json, ldap, 	libxml, mbstring, mysqli, mysqlnd, openssl, pcre, pdo_mysql, posix, readline, session, shmop, sockets, sodium, standard, 	sysvmsg, sysvsem, sysvshm, tokenizer, wddx, xml, xmlreader, xmlrpc, xmlwriter, xsl, zip, zlib) Setup: max_execution_time="36000" memory_limit="512M" post_max_size="1000M" safe_mode="" session.save_handler="files" 	upload_max_filesize="900M"  Software: Apache/2.4.29 (Ubuntu) (Apache/2.4.29 (Ubuntu) Server at help.rdn.local Port 80) 	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.102 Safari/537.36 Server Software: (Ubuntu) 	Server Version: 5.7.31-0ubuntu0.18.04.1 	Server SQL Mode:  	Parameters: glpi@localhost/glpi 	Host info: Localhost via UNIX socket 	 PHP version is at least 7.2.0 - Perfect! Sessions support is available - Perfect! Allocated memory > 64 Mio - Perfect! mysqli extension is installed ctype extension is installed fileinfo extension is installed json extension is installed mbstring extension is installed iconv extension is installed zlib extension is installed curl extension is installed gd extension is installed simplexml extension is installed intl extension is installed ldap extension is installed apcu extension is installed Zend OPcache extension is installed xmlrpc extension is installed CAS extension is installed exif extension is installed zip extension is installed bz2 extension is installed sodium extension is installed Database version seems correct (5.7.31) - Perfect! Timezones seems loaded in database The log file has been created successfully. Write access to /var/www/help/config has been validated. Write access to /var/www/help/files has been validated. Write access to /var/www/help/files/_dumps has been validated. Write access to /var/www/help/files/_sessions has been validated. Write access to /var/www/help/files/_cron has been validated. Write access to /var/www/help/files/_graphs has been validated. Write access to /var/www/help/files/_lock has been validated. Write access to /var/www/help/files/_plugins has been validated. Write access to /var/www/help/files/_tmp has been validated. Write access to /var/www/help/files/_cache has been validated. Write access to /var/www/help/files/_rss has been validated. Write access to /var/www/help/files/_uploads has been validated. Write access to /var/www/help/files/_pictures has been validated. Write access to /var/www/help/marketplace has been validated. Web access to files directory is protected

GLPI_ROOT: /var/www/help GLPI_CONFIG_DIR: /var/www/help/config GLPI_VAR_DIR: /var/www/help/files GLPI_MARKETPLACE_DIR: /var/www/help/marketplace GLPI_USE_CSRF_CHECK: 1 GLPI_CSRF_EXPIRES: 7200 GLPI_CSRF_MAX_TOKENS: 100 GLPI_TELEMETRY_URI: https://telemetry.glpi-project.org GLPI_INSTALL_MODE: TARBALL GLPI_NETWORK_MAIL: glpi@teclib.com GLPI_NETWORK_SERVICES: https://services.glpi-network.com GLPI_MARKETPLACE_PRERELEASES:  GLPI_USER_AGENT_EXTRA_COMMENTS:  GLPI_AJAX_DASHBOARD: 1 GLPI_CALDAV_IMPORT_STATE: 0 GLPI_DEMO_MODE: 0 GLPI_FORCE_EMPTY_SQL_MODE: 1 GLPI_DOC_DIR: /var/www/help/files GLPI_CACHE_DIR: /var/www/help/files/_cache GLPI_CRON_DIR: /var/www/help/files/_cron GLPI_DUMP_DIR: /var/www/help/files/_dumps GLPI_GRAPH_DIR: /var/www/help/files/_graphs GLPI_LOCAL_I18N_DIR: /var/www/help/files/_locales GLPI_LOCK_DIR: /var/www/help/files/_lock GLPI_LOG_DIR: /var/www/help/files/_log GLPI_PICTURE_DIR: /var/www/help/files/_pictures GLPI_PLUGIN_DOC_DIR: /var/www/help/files/_plugins GLPI_RSS_DIR: /var/www/help/files/_rss GLPI_SESSION_DIR: /var/www/help/files/_sessions GLPI_TMP_DIR: /var/www/help/files/_tmp GLPI_UPLOAD_DIR: /var/www/help/files/_uploads GLPI_NETWORK_REGISTRATION_API_URL: https://services.glpi-network.com/api/registration/ GLPI_MARKETPLACE_PLUGINS_API_URI: https://services.glpi-network.com/api/glpi-plugins/ GLPI_I18N_DIR: /var/www/help/locales GLPI_VERSION: 9.5.1 GLPI_SCHEMA_VERSION: 9.5.2 GLPI_MIN_PHP: 7.2.0 GLPI_YEAR: 2020

htmlawed/htmlawed version 1.2.5 in (/var/www/help/vendor/htmlawed/htmlawed) phpmailer/phpmailer version 6.1.6 in (/var/www/help/vendor/phpmailer/phpmailer/src) simplepie/simplepie version 1.5.5 in (/var/www/help/vendor/simplepie/simplepie/library) tecnickcom/tcpdf version 6.3.5 in (/var/www/help/vendor/tecnickcom/tcpdf) michelf/php-markdown in (/var/www/help/vendor/michelf/php-markdown/Michelf) true/punycode in (/var/www/help/vendor/true/punycode/src) iamcal/lib_autolink in (/var/www/help/vendor/iamcal/lib_autolink) sabre/dav in (/var/www/help/vendor/sabre/dav/lib/DAV) sabre/http in (/var/www/help/vendor/sabre/http/lib) sabre/uri in (/var/www/help/vendor/sabre/uri/lib) sabre/vobject in (/var/www/help/vendor/sabre/vobject/lib) laminas/laminas-cache in (/var/www/help/vendor/laminas/laminas-cache/src) laminas/laminas-i18n in (/var/www/help/vendor/laminas/laminas-i18n/src) laminas/laminas-serializer in (/var/www/help/vendor/laminas/laminas-serializer/src) monolog/monolog in (/var/www/help/vendor/monolog/monolog/src/Monolog) sebastian/diff in (/var/www/help/vendor/sebastian/diff/src) elvanto/litemoji in (/var/www/help/vendor/elvanto/litemoji/src) symfony/console in (/var/www/help/vendor/symfony/console) scssphp/scssphp in (/var/www/help/vendor/scssphp/scssphp/src) laminas/laminas-mail in (/var/www/help/vendor/laminas/laminas-mail/src/Protocol) laminas/laminas-mime in (/var/www/help/vendor/laminas/laminas-mime/src) rlanvin/php-rrule in (/var/www/help/vendor/rlanvin/php-rrule/src) blueimp/jquery-file-upload in (/var/www/help/vendor/blueimp/jquery-file-upload/server/php) ramsey/uuid in (/var/www/help/vendor/ramsey/uuid/src) psr/log in (/var/www/help/vendor/psr/log/Psr/Log) psr/simple-cache in (/var/www/help/vendor/psr/simple-cache/src) mexitek/phpcolors in (/var/www/help/vendor/mexitek/phpcolors/src/Mexitek/PHPColors) guzzlehttp/guzzle in (/var/www/help/vendor/guzzlehttp/guzzle/src) wapmorgan/unified-archive in (/var/www/help/vendor/wapmorgan/unified-archive/src) paragonie/sodium_compat in (/var/www/help/vendor/paragonie/sodium_compat/src) phpCas version 1.3.3 in (/usr/share/php)

`
```
**Additional context**

Migrated 9.4.2 to 9.5.1 nightly build ( 9.5.2)

**Comment by cedric-anne** [comment]
Hi,

It should have been fixed in latest nightly build (fixed by #7969).

Regards

**Comment by mYsn1k** [comment]
Update to 9.5-bugfixes.a50ae4b. solved problem. Thx.
